src/components/Header.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 const Header = () => {
   return (
-    <div className="h-20 grid gap-4 grid-cols-2 bg-rose-100 text-slate-700">
```

Comment:
This isn't grade school or our first html project. We don't need a rose pink header

Reply:
Definitely testing a color, chill

Reply:
lmao I was the one who did the pink

Reply:
The point stands even taller!

Reply:
I was making fun of myself with this one. I liked the pink but it doesn't scream professional